src/fluree/db/api/transact.cljc

```diff
@@ -13,6 +13,14 @@
     (sparql/->fql txn)
     txn))
 
+(defn insert
+  [db txn override-opts]
+  (go-try
+    (let [context    (or (ctx-util/txn-context txn)
+                         (:context override-opts))
+          parsed-txn {:insert (parse/jld->parsed-triples txn nil context)}]
```

Comment:
Why not use the new `jld->parsed-triples` function directly here?

Reply:
It is using that function, but I might misunderstand what you mean.

Reply:
sorry, i didn't see that. i just noticed that you constructed a new map with just an insert key instead of using the parsed triples. i thought you had done this for parsing, but now i see it's to be compatible with the `stage-triples` function. perhaps we can add a function in a later refactor to take the triples directly without needing the map, but it isn't a big deal for now.

Reply:
> perhaps we can add a function in a later refactor to take the triples directly without needing the map, but it isn't a big deal for now.

Currently `-stage-txn` protocol expects the map syntax, so this would likely involve adding another protocol method if we were to change the input format of the current staging process.